styles/modules.css

```diff
@@ -67,7 +67,9 @@
   margin-top: .2em;
 }
 
-@media only screen and (min-width: 690px) {
+/* Put in a break at 860 for header? */
+
```

Comment:
Still getting menu overlap because the nav is absolutely positioned.  Another break point here to set the max-width of the nav might fix this.